src/utils/update/hash.helper.ts

```diff
@@ -0,0 +1,14 @@
+import * as crypto from 'crypto';
+
+export function encode(input: string): string {
+
+  const salt = ""; // would load this from env variable
```

Comment:
Just leaving a note here to load this via env variable (if thats the route we decide to go)